File: Iniciante/URI 1041.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var x = parseFloat(line[0]);
var y = parseFloat(line[1]);

if (x == 0 && y == 0) {
    console.log('Origem');
} else if (x == 0) {
    console.log('Eixo Y');
} else if (y == 0) {
    console.log('Eixo X');
} else if (y > 0 && x > 0) {
    console.log('Q1');
} else if (y > 0 && x < 0) {
    console.log('Q2');
} else if (y < 0 && x < 0) {
    console.log('Q3');
} else if (y < 0 && x > 0) {
    console.log('Q4');
}

File: Iniciante/URI 1041.dart
import 'dart:io';

void main() {
  List<String> input = stdin.readLineSync().split(" ");
  double x = double.parse(input[0]);
  double y = double.parse(input[1]);
  if (x == 0 && y == 0) {
    print("Origem");
  } else if (x == 0) {
    print("Eixo Y");
  } else if (y == 0) {
    print("Eixo X");
  } else if (y > 0 && x > 0) {
    print("Q1");
  } else if (y > 0 && x < 0) {
    print("Q2");
  } else if (y < 0 && x < 0) {
    print("Q3");
  } else if (y < 0 && x > 0) {
    print("Q4");
  }
}
